compiler/fir/resolve/src/org/jetbrains/kotlin/fir/types/TypeUtils.kt

```diff
@@ -239,6 +239,9 @@ fun FirTypeRef.withReplacedConeType(
     }
 }
 
+fun FirTypeRef.withNullability(nullability: ConeNullability): FirTypeRef =
```

Comment:
Replacing nullability of `FirTypeRef` doesn't makes any sense, because type ref has no nullability. I rewrote changes in `transformElvisExpression` so they looks more idiomatic